source/community/reactnative/src/components/containers/container-wrapper.js

```diff
@@ -99,10 +100,25 @@ export class ContainerWrapper extends React.PureComponent {
 
             // bleed
             if (this.payload.bleed && this.props.containerStyle) {
-                computedStyles.push({ marginHorizontal: -padding });
+                const { isFirst, isLast } = this.props;
+                const marginLeft = isFirst ? -padding : 0;
```

Comment:
is -padding intended to make the padding value as negative number or inverse the sign?
if former, -padding will make negative number into positive. is that ok?

Reply:
We are constructing the padding value depends on the Hostconfig. So it should be ok.

const padding = hostConfig.getEffectiveSpacing(Enums.Spacing.Padding);